packages/react-components/src/index.css

```diff
@@ -15,7 +15,18 @@
     --o-ui-focus-ring-inset-lg: 0 0 0 var(--o-ui-focus-ring-thickness-lg) var(--o-ui-focus-ring-color);
     --o-ui-focus-ring-outset-inner-offset: 1px;
     --o-ui-focus-ring-outset: 0 0 0 var(--o-ui-focus-ring-outset-outer-offset) var(--o-ui-global-white), 0 0 0 calc(2px + var(o-ui-focus-ring-outset-outer-offset)) var(--o-ui-focus-ring-color);
-    --o-ui-focus-ring-transition: all .2s ease-in;
+    --o-ui-focus-ring-transition: all var(--o-ui-timing-1) var(--o-ui-easing-focus);
+    /* TRANSITIONS */
+    /* TIMING */
+    --o-ui-timing-1: .1s;
+    --o-ui-timing-2: .2s;
+    --o-ui-timing-3: .3s;
+    --o-ui-timing-4: .5s;
+    --o-ui-timing-5: .8s;
```

Comment:
Personally, I prefer the term "duration" to "timing". Also "timing" is still not precise enough for me, maybe something like "easing-duration" or "easing-timing" if you want to keep "timing".